core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,670 @@
+    height: auto;
+    width: 100%;
   }
-  .oppia-about-page-container .oppia-about-otter {
-    width: 200px;
+  .about-page .oppia-about-page-section-four {
+    background-image: url(/assets/images/about/background_dsk_1.svg);
+    background-size: 100% 100%;
+    padding: 6.5vh 2vw 10vw 0;
   }
-  .oppia-about-page-container .oppia-about-otter-container {
-    bottom: -2.3em;
-    display: inline-block;
-    position: absolute;
+  .oppia-about-page-section-four .oppia-about-page-section-four-left {
+    width: 45vw;
   }
-  .oppia-about-page-container .oppia-about-tab-text {
-    margin-top: -10px;
+  .oppia-about-page-section-four .oppia-about-lesson-image img,
+  .oppia-about-page-section-five .oppia-about-library-image img {
+    border: 4px solid #f9f9f9;
+    border-radius: 5px;
+    width: 100%;
   }
-  .oppia-about-page-container .oppia-license-link {
-    margin-top: -80px;
-    padding-top: 70px;
+  .oppia-about-page-section-four .oppia-about-classroom-button-container,
+  .oppia-about-page-section-five .oppia-about-classroom-button-container {
+    float: left;
   }
-  @media only screen and (max-width: 768px) {
-    .oppia-about-page-container .oppia-about-otter-container {
-      bottom: -1.8em;
-      left: -3em;
-    }
-    .oppia-about-page-container .oppia-about-otter {
-      width: 200px;
-    }
+  .oppia-about-page-section-five {
+    height: auto;
+    padding: 6.5vh 2vw 10vw 0;
+  }
+  .oppia-about-page-section-six {
+    background-image: url(/assets/images/about/background_dsk_2.svg);
+    background-size: 100% 100%;
+    padding: 6.5vh 2vw 10vw 0;
   }
-  .oppia-about-page-container .oppia-about-otter-speech-card {
+  .oppia-about-page-section-six .oppia-about-section-six-right {
     float: right;
-    width: 65%;
+    padding-left: 2vw;
+    padding-right: 2vw;
+    width: 50%;
+  }
+  .oppia-about-page-section-six .oppia-about-book-image img {
+    float: left;
+    height: auto;
+    padding-left: 4vw;
+    width: 45%;
+  }
+  .about-page .oppia-about-page-section-seven {
+    flex-direction: column;
+    height: auto;
+    padding: 6.5vh 0 5vw 0;
+    text-align: center;
+  }
+  .oppia-about-page-section-seven .oppia-get-started-title {
+    padding-bottom: 5vh;
+    width: 100%;
+  }
+  .oppia-about-page-section-four .oppia-about-button,
+  .oppia-about-page-section-five .oppia-about-button,
+  .oppia-about-page-section-six .oppia-about-button {
+    margin-top: 15px;
   }
-  @media only screen and (max-width: 500px) {
-    .oppia-about-page-container .oppia-about-otter-speech-card {
+  .about-page .oppia-about-button {
+    background-color: #015c53;
+    border-radius: 0;
+    color: #fff;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    margin-left: 0;
+    margin-top: -20px;
+    min-width: 250px;
+    padding: 10px;
+    text-transform: uppercase;
+    width: 220px;
+  }
+  .oppia-about-page-section-seven .oppia-about-button {
+    width: 510px;
```

Comment:
We are expanding the width here because of the long text in the button? If yes, then use min-with in the top button style and left other button width expand based on their text size

Reply:
we don't epand the button because of text, the button in last section is supposed to be that way only acc to mocks.